torch/csrc/autograd/python_variable.cpp

```diff
@@ -516,10 +516,9 @@ void initTensorImplConversion(PyObject* module) {
   });
   // set on the module level to avoid mixing pybind and plain CPython extensions
   m.def("_tensor_impl_raw_handle", [](torch::autograd::Variable* t) -> void* {
-    auto p = t->data().getIntrusivePtr();
     // We return a raw non-owning pointer here, we rely on surrounding
     // code to keep the original tensor alive
-    return p.get();
+    return t->getIntrusivePtr().get();
```

Comment:
The purpose of `_tensor_impl_raw_handle` is to obtain a handle to the data tensor's TensorImpl, so that we can call `blob._wrap_tensor_impl(handle)` to convert it to a Caffe2 Blob. Since after this patch, the Variable `t` itself is the data tensor, we can remove `.data()` here and use `t` directly.